src/ensembl/src/content/app/entity-viewer/types/protein.ts

```diff
@@ -0,0 +1,6 @@
+import { Product } from './product';
+
+export type Protein = {
+  transcriptId: string;
+  product: Product;
```

Comment:
this is a very strange combination of fields. A protein is a product. Why does it need the transcript id?

Reply:
I've removed the `Protein` object itself. CDM's `Product` seems to do the job.

Reply:
I forgot to remove `protein.ts` even though I switched to using `product.ts`. I've removed it in the latest commit. Sorry for the confusion.